packages/core/src/Transform.ts

```diff
@@ -719,7 +719,17 @@ export class Transform extends Component {
 
   private _translate(translation: Vector3, relativeToLocal: boolean = true): void {
```

Comment:
Update `translate` comments with "Translate by x along the x axis, y along the y axis, and z along the z axis."

Reply:
Done.